src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -313,7 +313,7 @@ class SummaryResponse(BaseSummaryResponse):
     def __init__(self, raw_response):
         super().__init__(raw_response)
 
-        _sum = raw_response['aggregations']['by_type']
+        _sum = raw_response['aggregations']['fileFormat']["myTerms"]
```

Comment:
Make sure this is referencing the sum of the file sizes and not the number of files.